lib/src/core/add_ordinal_suffix/add_ordinal_suffix.dart

```diff
@@ -6,13 +6,15 @@
 /// addOrdinalSuffix('پنج'); // پنجم
 /// ```
 String addOrdinalSuffix(String number) {
-  if (number.endsWith('ی')) {
-    return number + ' اُم';
+  if (number.endsWith('سی')) {
+    number = '$number ' 'اُم';
+  } else if (number.endsWith('سه')) {
+    var customNumber = number.substring(0, number.length - 2);
+    number = '$customNumber' 'سوم';
```

Comment:
For more consistency I recommend you to convert this 2 line to single line. Something like this.
```dart
number = '${number.substring(0, number.length - 2)} سوم';
```